.github/workflows/scala.yml

```diff
@@ -11,10 +11,11 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
-    name: Java ${{matrix.java}}
+    name: Test with Scala ${{ matrix.scala }} and Java ${{ matrix.java }})
     strategy:
       matrix:
         java: ['temurin:1.11', 'temurin:1.17', 'temurin:1.21']
+        scala: [ 2.13.13, 3.3.1 ]
```

Comment:
We should use latest version from LTS, which is [3.3.3](https://github.com/scala/scala3/releases/tag/3.3.3)